source/OptimaJet.DWKit.StarterApplication/Controllers/UsersController.cs

```diff
@@ -7,12 +7,14 @@
 using Microsoft.AspNetCore.Authentication.JwtBearer;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
```

Comment:
why are these here?

Reply:
May have needed them at one time.  I hoped that Visual Studio would visually show unused namespaces.

Reply:
All three of those namespaces are needed.

Reply:
I submitted a commit that remove unnecessary using statements.